src/__configuration__/roadmap/index.tsx

```diff
@@ -17,201 +17,3 @@ export type RoadmapBucket = {
     applies?: FrameworkFilter[];
     items: RoadmapItem[];
 };
-
```

Comment:
Since this file is just type definitions now, we should move it out of the configuration folder and into the types area.

Reply:
Yep this makes sense, I've moved the Roadmap types into the type3s area.